### PR TITLE
Week 16, Project Labyrinth - Therese H. & Noemi F. 

### PR DESCRIPTION
**Pull request by ThereseHag**
Link to my deployed project:

https://therese-and-noemi-labyrinthgame.netlify.app/

My reflections on how this week's project turned out:

It was awesome. Team work was great. We faced some challenges but they didn't last for long, and it was always a joy to be able to find and fix them. It was a good project since it made you learn a lot, since there were so many things to take into consideration in order to make everything work.

Things I'd like to have clarified or explained in more detail:

Nothing really. There might be some more things we would have added if we had more time, but we feel we know how to do it, in case we want to go back to this at a later point.

### CODE REVIEW

**Review by esteficodes — APPROVED** [review]
Hi Therese and Noemi! Great job at coding this labyrinth! 🌈🙂

Going through your code I noticed that it's solid and neat. I didn't even find any console logs left. I really like the way you spiced it up with styled-components and even a Lottie animation. Way to go!
I can see that your project fulfills all the basic requirements because it's responsive, uses Redux to store the current state of the game, handles the response delay with a beautiful loading animation and even uses thunks to wrap the API calls.
To me, this is an approved project. Good job! 🙋‍♀️

File: code/src/components/FinalPage.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const Title = styled.h1`
+  padding: 0 15px;
+  font-size: 25px;
+  color: #C4007D;
+  text-align: center;
+`
+const FinalPage = () => {
+
+  return (
+    <>
```
Comment (KarolCislowski) [inline code/src/components/FinalPage.js:13]: You have only 1 component here so you shouldn't use react fragment.

File: code/src/components/StartScreen.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  const dispatch = useDispatch();
+
+  const onUserNameSet = () => {
+    dispatch(games.actions.setUserName(inputValue));
+    dispatch(startGame(inputValue));
+  };
+
+  return (
+    <>
+      <Container>
```
Comment (KarolCislowski) [inline code/src/components/StartScreen.js:81]: the wrong usage of react fragment, again 😿

File: code/src/components/StartScreen.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        <IntroductionText>
+          Type your name to enter the labyrinth...
+        </IntroductionText>
+        <Input
+          type="text"
+          placeholder="Enter your name to start"
+          value={inputValue}
+          onChange={(event) => setInputValue(event.target.value)}
+          required
+        />
```
Comment (KarolCislowski) [inline code/src/components/StartScreen.js:92]: It is not nice that you forgot about the form tag. It striped your input from submission on enter, what is the most used sending the form method 😭

File: code/src/components/Game.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  );
+
+  const dispatch = useDispatch();
+
+  return (
+    <Wrapper>
+      <LoadingContainer>
+        <DescriptionTitle>{description}</DescriptionTitle>
+      </LoadingContainer>
+      {steps.length !== 0 ? (
```
Comment (KarolCislowski) [inline code/src/components/Game.js:91]: great solution 😻

**Review by KarolCislowski — COMMENTED** [review]
Great App 😻

**Pull request closed without merging**